src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -122,7 +122,7 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                     if (addresses.SingleOrDefault() != "http://localhost:5000")
                     {
                         var joined = string.Join(", ", addresses);
-                        throw new NotSupportedException($"Specifying address(es) '{joined}' is incompatible with also configuring endpoint(s) in UseKestrel.");
+                        _logger.LogWarning($"Address(es) '{joined}' were overridden. Binding to endpoints defined in UseKestrel instead.");
```

Comment:
nit: `nameof(UseKestrel)`
nit 2: add `()` after `UseKestrel` to make it clearer it's referring to the method call